.github/workflows/ci.yml

```diff
@@ -62,6 +62,9 @@ jobs:
 
         steps:
             - uses: actions/checkout@v4
+              with:
+                fetch-depth: 0
+                fetch-tags: true
             - name: make ${{ matrix.variant }}
```

Comment:
not sure what the expected behavior here is:

```console
$ git clone --depth=0 git@github.com:docker/docker-py.git
fatal: depth 0 is not a positive number
```

Reply:
it's a value that's valid for the gh action, it just means fetch all essentially

Reply:
https://github.com/actions/checkout for reference